### PR TITLE
Fixes #19 - Added new device types in the 1.3 specification

### LINKED ISSUE #19: Updates to include new Device Types in Matter 1.3
**Why is this content needed**
Once Matter 1.3 is launched the details here will be out of date

**Who will benefit,**
External people looking at Matter will know what is now supported.

**Are you able and willing to create the content**
Happy to add content for the EM TT

**Additional context**
None

### PR DESCRIPTION
**Pull request by jamesharrow**
Added new device types in the 1.3 specification (from 0.9 ballot). Note that not sure if Water Freeze Detector and Rain Sensor are supported and certifiable. Needs checking.

### CODE REVIEW

**Comment by ReneJosefsen** [comment]
Given the clusters used by the two mentioned device type did pass SVE, I'm not aware that these should NOT be a part of 1.3?

File: content/HowItWorks/DeviceTypes.md [hunk 1]
```diff
@@ -6,25 +6,31 @@ weight = 10
...
+  - [OTA Provider](#ota-provider)
+  - [Bridged Node](#bridged-node)
+  - [Electrical Sensor](#electrical-sensor)
+  - [Device Energy Management](#device-energy-management)
+- [Application Device Types](#application-device-types)
   - [On/Off Light](#onoff-light)
   - [Dimmable Light](#dimmable-light)
   - [Color Temperature Light](#color-temperature-light)
   - [Extended Color Light](#extended-color-light)
-- Smart Plugs/Outlets and Other Actuators
```
Comment (ReneJosefsen) [inline content/HowItWorks/DeviceTypes.md:16]: Is it intentional to remove these headings?

Comment (jamesharrow) [reply content/HowItWorks/DeviceTypes.md:16]: No! - I think my editor did that - I'll revert.

File: content/HowItWorks/DeviceTypes.md [hunk 2]
```diff
@@ -6,110 +6,178 @@ weight = 10
...
+    - [Light Sensor](#light-sensor)
+    - [Occupancy Sensor](#occupancy-sensor)
+    - [Temperature Sensor](#temperature-sensor)
+    - [Pressure Sensor](#pressure-sensor)
+    - [Flow Sensor](#flow-sensor)
+    - [Humidity Sensor](#humidity-sensor)
+    - [On/Off Sensor](#onoff-sensor)
+    - [Smoke CO Alarm](#smoke-co-alarm)
+    - [Water Freeze Detector](#water-freeze-detector)
+    - [Water Leak Detector](#water-leak-detector)
```
Comment (jamesharrow) [inline content/HowItWorks/DeviceTypes.md:48]: Are the water Freeze Detector and Rain Sensor certifiable device types? It looks to me that they use the same clusters (Boolean State) as Water Leak detector ? So did these therefore also get through 1.3?

**Comment by jamesharrow** [comment]
> Given the clusters used by the two mentioned device type did pass SVE, I'm not aware that these should NOT be a part of 1.3?

I assume we are talking about Water Freeze Detector and Rain Sensor.

As you say these use the same underlying cluster (Boolean State) so I'm assuming that they are, but Rob's email for 0.9 ballot didn't explicitly mention them, but did mention Water Leak detector.

File: content/HowItWorks/DeviceTypes.md (hunk 2 above)
Comment (ReneJosefsen) [reply content/HowItWorks/DeviceTypes.md:48]: That is certainly my impression, we are testing clusters at certification and not device types (for now) and the clusters did land as a part of 1.3.

**Comment by ReneJosefsen** [comment]
> > Given the clusters used by the two mentioned device type did pass SVE, I'm not aware that these should NOT be a part of 1.3?
> 
> I assume we are talking about Water Freeze Detector and Rain Sensor.
> 
> As you say these use the same underlying cluster (Boolean State) so I'm assuming that they are, but Rob's email for 0.9 ballot didn't explicitly mention them, but did mention Water Leak detector.

I can not speak to the reason for that, but I would assume it is a simple oversight. The main focus was on the leakage detector, the water freeze detector and rain sensor was "bolted on" during the development since they were obvious device types that could easily be constructed using the proposed clusters, so often the water management effort was referred to as just leakage and valve 🤷🏻‍♂️ I'm not sure who can give the final answer here 🤔 @robszewczyk do you have any input here?

**Comment by Videonisse** [comment]
I assume you have already sorted this out, but anyway, they are in the Device Library spec for v1.3:

- 7.11. Water Freeze Detector Device Type
- 7.13. Rain Sensor Device Type.

**Pull request merged**